src/server.js

```diff
@@ -168,6 +168,30 @@ server
             </script>`
             : ''
         }
+        ${
+          runtimeConfig.MATOMO_URL && runtimeConfig.MATOMO_SITE_ID
+            ? `
+            <!-- Matomo -->
+            <script type="text/javascript">
+                var _paq = window._paq || [];
+                /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+                _paq.push(['trackPageView']);
```

Comment:
I think these should be configureable, as if I remember correctly, there's multiple one can add or not have.

Reply:
I didn't exactly know what this image tracking was and it didn't allow it in the head tag.
I just read how it works and it's basically what you are mentioning here. 
I'm going to make a new pull request, already tested and it works on edge with javascript disabled (not allowed)